css-color-adjust-1/Overview.bs

```diff
@@ -24,13 +24,27 @@ spec:css-color-4; type:property; text:color
 spec:fill-stroke-3; type:property;
 	text:fill
 	text:stroke
+spec: webdriver; urlPrefix: https://w3c.github.io/webdriver/#; type: dfn;
+	text: error; url: dfn-error
+	text: error code; url: dfn-error-code
+	text: extension command; url: dfn-extension-commands
+	text: extension commands; url: dfn-extension-commands
+	text: getting a property; url: dfn-getting-properties
+	text: invalid argument; url: dfn-invalid-argument
+	text: null; url: dfn-null
+	text: remote end steps; url: dfn-remote-end-steps
+	text: session; url: dfn-session
+	text: success; url: dfn-success
+	text: undefined; url: dfn-undefined
```

Comment:
The links for "undefined" and "null" do not work, but I am not sure why. The rest of them are working as expected.

Reply:
@tabatkins perhaps you can help with this?

Reply:
Ah, you're using `anchors` values (defining brand new anchors that don't exist in the linking database), but this is a `link-defaults` block (declaring defaults for your autolinks).

If all the others work, then you don't need them at all (or possibly they're ambiguous and these lines, interpreted as link-defaults, are fixing their ambiguity).

These terms in particular, tho, aren't WebDriver definitions at all, they're from the ECMAScript spec. `undefined` is defined properly, use `{{globalThis/undefined}}` to link it. (You can find this by running `bikeshed refs --text=undefined`.) `null` doesn't seem to have a definition, for some reason, I suppose that'll have to get a `<pre class=anchors>` line.

Reply:
That did the trick, thanks @tabatkins!

And you were right, some of those weren't required. I removed the ones that were unambiguous, updated `undefined` to use `{{globalThis/undefined}}`, and added `<pre class=anchors>` for `null`.